New tracing section in configuration (#692)
kiali/kiali.io: content/en/docs/Configuration/p8s-jaeger-grafana/tracing/jaeger.md

---
title: "Jaeger"
description: >

>   This page describes how to configure Jaeger for Kiali.

Reviewer: Remember this description shows up in the subtitle, 

![image](https://github.com/kiali/kiali.io/assets/2029470/1195bc6e-36a8-47cd-bd2c-a1fbe49da0a6)

so fix this copy-n-paste typo:

```suggestion
  This page describes how to configure Grafana Tempo for Kiali.
```

If this endpoint is unreachable, Kiali will disable features that use
distributed tracing data.

If your Jaeger instance has a different service name or is installed to a
different namespace, you must manually provide the endpoint where it is
available, like in the following example:

```yaml
spec:
  external_services:
    tracing:
      # Enabled by default. Kiali will anyway fallback to disabled if
      # Jaeger is unreachable.
      enabled: true
      # Jaeger service name is "tracing" and is in the "telemetry" namespace.
      # Make sure the URL you provide corresponds to the non-GRPC enabled endpoint
      # if you set "use_grpc" to false.
      in_cluster_url: "http://tracing.telemetry:16685/jaeger"
      use_grpc: true
      # Public facing URL of Jaeger
      url: "http://my-jaeger-host/jaeger"
```

Minimally, you must provide `spec.external_services.tracing.in_cluster_url` to
enable Kiali features that use distributed tracing data. However, Kiali can
provide contextual links that users can use to jump to the Jaeger console to
inspect tracing data more in depth. For these links to be available you need to

> set the `spec.external_services.tracing.url` which may mean that you should

Reviewer: If you set this, it isn't optional ("may mean"... you "should")... it is required that you expose Jaeger externally. Otherwise, that URL will be invalid. So be more definitive:
```suggestion
set the `spec.external_services.tracing.url` to the URL where you
```

Author: That's right, it is required, thanks for the suggestion!
